charts/harvester-cloud-provider/Chart.yaml

```diff
@@ -31,7 +31,7 @@ annotations:
   catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.27.0-0'
   catalog.cattle.io/release-name: harvester-cloud-provider
   catalog.cattle.io/os: linux
-  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.9.0-0'
```

Comment:
please set `< 2.10.0-0`  to support rancher v2.9 as well

Reply:
Updated it. Thanks.